contracts/protocol/facets/SellerHandlerFacet.sol

```diff
@@ -56,7 +56,9 @@ contract SellerHandlerFacet is SellerBase {
      *         All other fields should be filled, even those staying the same.
      * @dev    Active flag passed in by caller will be ignored. The value from storage will be used.
      *
-     * Emits a SellerUpdated event if successful.
+     * Emits a SellerUpdateApplied event if the seller has changed the treasury.
+     * Emits a SellerUpdatePending event if the seller has requested an update for admin, clerk, operator, or auth token.
+     * Addresses owner of new values for admin, clerk, operator or auth token need to opt-in to update.
```

Comment:
Revised to make a little clearer.

_Holder of new auth token and/or owner(s) of new addresses for admin, clerk, operator must opt-in to the update._

Reply:
done